src/sknnr/_gnn_rda.py

```diff
@@ -0,0 +1,48 @@
+from sklearn.base import TransformerMixin
+
+from ._base import TransformedKNeighborsRegressor, YFitMixin
+from .transformers import RDATransformer
+
+
+class GNNRDARegressor(YFitMixin, TransformedKNeighborsRegressor):
+    def __init__(self, n_components=None, **kwargs):
+        super().__init__(**kwargs)
+        self.n_components = n_components
+
+    def _get_transformer(self) -> TransformerMixin:
+        return RDATransformer(self.n_components)
+
+    def _more_tags(self):
+        unsupported_1d = "RDA requires 2D y arrays."
+
+        return {
+            "allow_nan": False,
+            "requires_fit": True,
+            "requires_y": True,
+            "multioutput": True,
+            "_xfail_checks": {
```

Comment:
I wasn't *exactly* sure what you meant here, so I haven't yet done anything here.  Are you suggesting that the dictionary is something like:
```
unsupported_1d = "Constrained ordination requires 2D y arrays."
UNSUPPORTED_1D_CHECKS = dict(
    check_estimator_dtypes=unsupported_1d,
    check_dtype_object=unsupported_1d,
    ...
)
```
and then in `GNNRegressor` and (now) `ConstrainedOrdination`, we do:
```
def _more_tags(self):
    return {
        "allow_nan": False,
        ...
        "_xfail_checks": UNSUPPORTED_1D_CHECKS
    }
```
I'm a bit confused because the check names aren't exactly the same between the two classes.

Reply:
That's what I had in mind. I think the primary difference between the two sets of checks was that different tests get collected for transformers vs. estimators and based on their tags, although I can't say I have a great understanding of that. It's probably not worth the complexity if we have to keep two separate sets of checks to xfail. We could just import a `dict` of shared checks and manually add the rest, but I'm not sure that's an improvement either. I'm fine to leave as-is for now, although I keep hoping we'll find a way to get rid of those some day...

Reply:
I agree that it's "unsightly" and it would be nice to squirrel it away somewhere that we didn't have it so present, but I think I'm on board with keeping it as-is until we can somehow magically pass all those checks.